microsoft/testsuites/network/sriov.py

```diff
@@ -401,9 +401,11 @@ def verify_sriov_ethtool_offload_setting(self, environment: Environment) -> None
         # iperfResults.log stored client side log
         source_iperf3 = server_node.tools[Iperf3]
         dest_iperf3 = client_node.tools[Iperf3]
-        source_iperf3.run_as_server()
+        source_iperf3.run_as_server_async()
         dest_iperf3.run_as_client_async(
-            server_node.internal_address, client_iperf3_log, 1800
+            server_ip=server_node.internal_address,
+            log_file=client_iperf3_log,
+            run_time_seconds=1800,
```

Comment:
It's a p2 test case, does it need to run so long?

Reply:
There are some testing after running iperf3, and after the testing, the iperf3 will be killed, so actual running time is less than 1800, just make sure that the iperf3 won't quit/interrupt during testing, I will double check the actual running time, then update it to align the actual time and add comments here, since last time you asked the same question.

Reply:
The time is terrible here, and I may ask later, after several months... One solution is to remove it totally, and leave it run forever. If some test cases forget to kill it, and cause some problem, we can find it soon.

Reply:
Each test case has its timeout, so it won't block test case timeout.

Reply:
updated